Sources/SPTDataLoaderServiceSessionSelector.m

```diff
@@ -98,6 +132,7 @@ - (void)invalidateAndCancel
 {
     [self.waitingSession invalidateAndCancel];
     [self.nonWaitingSession invalidateAndCancel];
+    [self.backgroundSession invalidateAndCancel];
```

Comment:
This should use the ivar, not the property. If up until this point, the background session had not been created, it will be created just to invalidate it.

Reply:
Oh that's true, thanks